example-cjs/e2e/api.spec.ts

```diff
@@ -48,36 +50,172 @@ describe('electron APIs', () => {
       expect(processType).toEqual('browser');
     });
   });
+});
+
+describe('mock', () => {
+  afterEach(async () => {
+    await browser.electron.removeMocks();
+  });
+
+  describe('setMock', () => {
+    it('should mock an electron API function', async () => {
+      const dialog = await browser.electron.mock('dialog');
+      await dialog.setMock('showOpenDialog');
```

Comment:
I wonder if it makes more sense to just call 
```ts
const mockedShowOpenDialog = await browser.electron.mock('dialog', 'showOpenDialog')
```
instead of doing this. It would more align to `vitest` and `jest` APIs so user are more familiar with what they need to do.

Is there any advantage of an additional `getMock` / `setMock` call?

Reply:
You would call `unmock` via:

```ts
browser.electron.unmock('dialog', 'showOpenDialog')
```

see https://vitest.dev/api/vi.html#vi-unmock

Reply:
I'll see what I can do here, does make sense to have a single-liner for the mock but there is a reason I did it this way.  I was running into trouble getting the mock value to update after being called - because it needs to get the calls from the inner mock in the electron main process and re-apply them.  I messed around with getters for a while but wasn't able to make the following work:

```
const mockedShowOpenDialog = await browser.electron.mock('dialog', 'showOpenDialog');
await browser.electron.execute(
  async (electron) =>
    await electron.dialog.showOpenDialog({
      title: 'my dialog',
      properties: ['openFile', 'openDirectory'],
    }),
);

expect(mockedShowOpenDialog).toHaveBeenCalledTimes(1);
expect(mockedShowOpenDialog).toHaveBeenCalledWith({
  title: 'my dialog',
  properties: ['openFile', 'openDirectory'],
});
```

So, `setMock` may be condensed into `mock` but a separate call to update the outer mock may well be unavoidable, unless there's another way of doing it that I'm not aware of...

Reply:
getting there...